.github/workflows/release_dev.yml

```
@@ -108,11 +128,17 @@ jobs:
           name: built-file
           path: ./build_output
 
+      - name: Debug tag and commit 2
+        run: |
```

actionlint (1.7.4)

property "tag_info" is not defined in object type {}

(expression)